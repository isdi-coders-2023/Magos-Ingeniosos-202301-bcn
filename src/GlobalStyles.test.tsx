import { render } from "@testing-library/react";
import GlobalStyles from "./GlobalStyles";

describe("Given a GlobalStyles component", () => {
  describe("When the document renders", () => {
    test("Then the document head should include those styles", () => {
      render(<GlobalStyles />);

      expect(document.head).toMatchSnapshot();
    });
  });
});
