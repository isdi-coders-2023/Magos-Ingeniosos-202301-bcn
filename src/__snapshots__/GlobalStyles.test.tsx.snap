// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a GlobalStyles component When the document renders Then it should have the styles defined in GlobalStyles 1`] = `<head />`;

exports[`Given a GlobalStyles component When the document renders Then the document head should include those styles 1`] = `<head />`;
